server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/shard/GetShardSnapshotRequest.java

```diff
@@ -102,4 +103,12 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(repositories, shardId);
     }
+
+    @Override
+    public String getDescription() {
+        final StringBuilder stringBuilder = new StringBuilder("shard").append(shardId).append(", repositories[");
+        Strings.collectionToDelimitedStringWithLimit(repositories, ",", "", "", 1024, stringBuilder);
+        stringBuilder.append("]");
```

Comment:
Would it be possible to chain the `append` and `toString` calls on the same line?

Reply:
It's possible, yes, but I would generally keep them separate like this. Doing it like this means that if you add to the description in future you can just add lines in between, rather than having to adjust the `return` statement, which simplifies reviews. Consider it as a low-level case of CQRS.